File: projects/m1/030-maximum-integer/solutions/enricozoboli/solution_maximum_integer.py
import random

update = 0
maximum = random.randint(1,100)
print(maximum)

for i in range(100):
    number = random.randint(1, 100)
    if number  > maximum:
        maximum = number
        update += 1
        print(f"{number} <== update")
    else:
        print(number)
        continue
       
print(f"\nThe maximum value found was {maximum}")
print(f"The maximum value was updated {update} times")

File: projects/m1/030-maximum-integer/solutions/valentinaBazzacchi/solution-maximum-integer.js
let maxNumber = Math.round(Math.random()*99)+1;
let display = maxNumber + " <-- First Number";
alert(display);
let currentNumber;
let counter = 0;

for (let i = 0; i < 100; i++) {
    currentNumber = Math.round(Math.random()*99)+1;
        
        if (currentNumber > maxNumber) {
        maxNumber = currentNumber;
        counter += 1;
        display = maxNumber + " <-- Update";
        alert("i = " + i + "\n" + display);
        
        } else {
        display = currentNumber;
        alert("i = " + i + "\n" + display);
        } // end else

} // end for  
alert("The maximum value found was: " + maxNumber + "\nThe maximum value was updated " + counter + " times");